validator/client/validator.go

```diff
@@ -420,7 +420,7 @@ func (v *validator) UpdateDuties(ctx context.Context, slot types.Slot) error {
 // to eagerly subscribe to subnets so that the aggregator has attestations to aggregate.
 func (v *validator) subscribeToSubnets(ctx context.Context, res *ethpb.DutiesResponse) error {
 	subscribeSlots := make([]types.Slot, 0, len(res.CurrentEpochDuties)+len(res.NextEpochDuties))
-	subscribeCommitteeIDs := make([]uint64, 0, len(res.CurrentEpochDuties)+len(res.NextEpochDuties))
+	subscribeCommitteeIDs := make([]types.CommitteeIndex, 0, len(res.CurrentEpochDuties)+len(res.NextEpochDuties))
```

Comment:
Maybe change to `subscribeCommitteeIndices`?

Reply:
Yeah we should avoid the word ID, as it is misleading. We should always use Index or Indices